problems/objects.js

```diff
@@ -61,7 +79,11 @@ const appleSet = () => {};
  *
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = (appleCountByName) => {
+  // Object.values(appleCountByName)
+  let sum = appleCountByName.Adam + appleCountByName.Eve
```

Comment:
This code can be refactored to one line -- Consider if we need the sum variable to return the number we want.